hibernate-core/src/test/java/org/hibernate/jpa/test/lock/QueryLockingTest.java

```diff
@@ -145,6 +145,7 @@ public void testNativeSql() {
 	}
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done